docs/pages/guides/devtools.mdx

```diff
@@ -0,0 +1,127 @@
+---
+meta:
+  title: "DevTools"
+  parentTitle: "Guides"
+  description:
+    "Learn how to leverage Liveblocks DevTools to inspect and troubleshoot your
+    collaborative application."
+---
+
+## What are Liveblocks DevTools?
+
+Our [DevTools](/devtools) are a browser-based extension that integrates with
+Liveblocks and your local development environment. This allows you to easily
+troubleshoot, debug, and optimize your collaborative online experiences.
+
+<Banner title="Live demo video">
+
+If you want to see how this works in video, be sure to check out this
+[live demo](https://www.youtube.com/watch?v=-6Jxejp0iuw) on YouTube.
+
+</Banner>
+
+## How DevTools work
+
+Our DevTools communicate with Liveblocks servers in development mode and
+retrieve the current state of storage and presence (and history is coming
+soon!). The DevTools support one-way data syncs from the Liveblocks client and
+are compiled away in production, similar to other DevTools like React and Redux.
```

Comment:
ok thank you . will make a proposition here.

Reply:
since I more or less removed this section, I will mark it as resolved.  I won't mention anything about history, but I do think we may get questions because it is obviously there, yet we can’t access it.